apiary.apib

```diff
@@ -3243,6 +3251,40 @@ To create a new feature, provide the following parameters as JSON in the request
         "name": "show-new-design",
         "type": "admin",
         "description": "Users with this feature will see new UI",
+        "canBeManageByAdmin": false,
+        "canBeManagedViaAPI": true,
+        "created": "2016-04-28T15:24:51.620Z"
+    }
+    ```
+
+### Update a feature [PATCH /manage/features/{id}]
+A partial update of a feature
+
++ Parameters
+    + id: 1 (required, number) - ID of the feature to update
+
++ Attributes
+
+    + description: Users with this feature will see new UI (optional) - Short description of the feature
+    + canBeManageByAdmin: false (boolean, optional) - If true, the feature can be assigned by a user without the superadmin role
+    + canBeManagedViaAPI: true (boolean, optional) - Enables or disables the ability to assign the project/admin feature using the API
+
++ Request (application/json)
+    + Headers
+
+            X-KBC-ManageApiToken: your_token
```

Comment:
cekal bych pro update nejaky smysluplny example (treba description), kdyz je to ten update :-)

Reply:
<img width="999" alt="Screenshot 2023-03-09 at 19 17 52" src="https://user-images.githubusercontent.com/6448364/224118846-31efcba5-60d9-4968-884e-04db68cde95a.png">
Ono v nahladu ta ukazka je, takto si to myslel nie ?

Reply:
jo, vlastne, ono se to tam vygeneruje z tech atributu 👍